modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/api/IgniteSqlImpl.java

```diff
@@ -536,14 +536,18 @@ public CompletableFuture<Void> executeScriptAsync(String query, @Nullable Object
         }
 
         try {
+            SqlProperties properties = SqlPropertiesHelper.newBuilder()
```

Comment:
it looks better to add default query timeout to org.apache.ignite.internal.sql.engine.SqlQueryProcessor#DEFAULT_PROPERTIES instead of add the timeout to all places

Reply:
Thanks. Done.